indico/modules/__init__.py

```diff
@@ -15,4 +15,7 @@
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
 from indico.modules.base import ModuleHolder, Module
-import indico.modules.categories  # TODO: remove this import once it's imported from somewhere else
+
+# TODO: remove imports below once they're imported from somewhere else:
+import indico.modules.categories
```

Comment:
Can't wait till we get rid of MaKaC, so that I start taking pre-commit PEP8 errors seriously...